src/panoptes/pocs/focuser/focuser.py

```diff
@@ -51,6 +52,7 @@ def __init__(self,
                  model='simulator',
                  port=None,
                  camera=None,
+                 timeout=5,
```

Comment:
Maybe keep the default as it was before?
```suggestion
                 timeout=2.0,
```

Reply:
We discussed with @AnthonyHorton that this can help a little with the birger issues we're having, as we wait a bit longer for the response coming from the birgers and thus avoid the `echo` error we've been finding.